Vagrantfile

```diff
@@ -121,6 +121,17 @@ echo "export https_proxy='$2'" >> /etc/profile.d/envvar.sh
 source /etc/profile.d/envvar.sh
 SCRIPT
 
+def customize(v, id)
+  # make all nics 'virtio' to take benefit of builtin vlan tag
+  # support, which otherwise needs to be enabled in Intel drivers,
+  # which are used by default by virtualbox
+  # changes the settings for the first 5 NICs, regardless of presence
+  1.upto(5) do |n|
+    v.customize ['modifyvm', :id, '--nictype' + n.to_s , 'virtio']
```

Comment:
If you use a doublequoted string + interpolation, #to_s will automatically be called on the interpolated object:

```rb
"--nictype#{n}"
```